distribution/feature-packs/server-feature-pack/src/main/resources/content/bin/migrate-domain-standalone.cli

```diff
@@ -47,6 +47,7 @@ if (result == undefined) of /profile=$standaloneProfile/subsystem=infinispan/cac
   echo Updating authorization cache container..
   /profile=$standaloneProfile/subsystem=infinispan/cache-container=keycloak/local-cache=authorization/component=eviction/:write-attribute(name=strategy,value=LRU)
   /profile=$standaloneProfile/subsystem=infinispan/cache-container=keycloak/local-cache=authorization/component=eviction/:write-attribute(name=max-entries,value=100)
+  echo
```

Comment:
What's with the empty echo statements here?

Reply:
That's just for adding another line to the output of the CLI command. I've just tried without echo and the output doesn't look so nice. Also the pattern with empty echo is used in whole file and all the previous instructions.

Reply:
Just to compare, here is the output file with empty echo command:
``
Removing eviction from keycloak users cache and replacing with object-memory
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}

Removing eviction from keycloak authorization cache and replacing with object-memory
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}

Removing eviction from keycloak keys cache and replacing with object-memory
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}

``

And here the one without echo:
``
Removing eviction from keycloak users cache and replacing with object-memory
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
Removing eviction from keycloak authorization cache and replacing with object-memory
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
Removing eviction from keycloak keys cache and replacing with object-memory
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
{
    "outcome" => "success",
    "response-headers" => {"process-state" => "reload-required"}
}
``

My vote is, that the first one looks better